src/CMakeLists.txt

```diff
@@ -412,16 +412,19 @@ if(USE_HEIF)
   endif()
 endif()
 
-#  For now we use the LibRaw submodule
-if (USE_LIBRAW)
-#    find_package(libraw 0.20.2)
-#    if (libraw_FOUND)
-#        list(APPEND LIBS ${libraw_LIBRARY})
-#        add_definitions(-DHAVE_LIBRAW=1)
-#        list(APPEND SOURCES "common/imageio_libraw.c")
-#    endif()
-   add_definitions(-DHAVE_LIBRAW=1)
-   list(APPEND SOURCES "common/imageio_libraw.c")
+# For now we use the LibRaw submodule
+if(USE_LIBRAW)
```

Comment:
Can we add some status message here:
1. saying the bundled libraw is being used
2. or saying that the OS libraw is being used

Reply:
Good idea, will do.

Btw, what should be the default? Prefer the system lib if available, or intree copy (as is currently)?

Reply:
I would stay on intree copy for now until we have more visibility on which distribs have updated LibRaw. And we could reconsider the default later.